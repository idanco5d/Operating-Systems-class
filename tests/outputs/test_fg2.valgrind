==8211== Memcheck, a memory error detector
==8211== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==8211== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==8211== Command: /mnt/c/Users/illus/os/hw/1/smash
==8211== Parent PID: 8206
==8211== 
==8211== 
==8211== FILE DESCRIPTORS: 4 open (3 std) at exit.
==8211== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_fg2.valgrind
==8211==    <inherited from parent>
==8211== 
==8211== 
==8211== HEAP SUMMARY:
==8211==     in use at exit: 0 bytes in 0 blocks
==8211==   total heap usage: 104 allocs, 104 frees, 83,725 bytes allocated
==8211== 
==8211== All heap blocks were freed -- no leaks are possible
==8211== 
==8211== For lists of detected and suppressed errors, rerun with: -s
==8211== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
