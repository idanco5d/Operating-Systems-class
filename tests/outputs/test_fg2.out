smash> smash> smash> smash> smash> smash> sleep 300& : 9057
smash: got ctrl-C
smash: process 9057 was killed
smash> sleep 100& : 9052
smash: got ctrl-C
smash: process 9052 was killed
smash> sleep 200& : 9055
smash: got ctrl-C
smash: process 9055 was killed
smash> sleep 400& : 9060
smash: got ctrl-C
smash: process 9060 was killed
smash> sleep 500& : 9062
smash: got ctrl-C
smash: process 9062 was killed
smash> smash: sending SIGKILL signal to 0 jobs:
